docs/index.md

```diff
@@ -1,45 +1,76 @@
+* **Application integration tests**: for running your application in a short-lived test mode with dependencies, such as databases, message queues or web servers.
+* **UI/Acceptance tests**: use containerized web browsers, compatible with Selenium, for conducting automated UI tests. Each test can get a fresh instance of the browser, with no browser state, plugin variations or automated browser upgrades to worry about. And you get a video recording of each test session, or just each session where tests failed.
+* **Much more!** Check out the various contributed modules or create your own custom container classes using [`GenericContainer`](features/creating_container.md) as a base.
 
- * **Data access layer integration tests**: use a containerized instance of a MySQL, PostgreSQL or Oracle database to test your data access layer code for complete compatibility, but without requiring complex setup on developers' machines and safe in the knowledge that your tests will always start with a known DB state. Any other database type that can be containerized can also be used.
- * **Application integration tests**: for running your application in a short-lived test mode with dependencies, such as databases, message queues or web servers.
- * **UI/Acceptance tests**: use containerized web browsers, compatible with Selenium, for conducting automated UI tests. Each test can get a fresh instance of the browser, with no browser state, plugin variations or automated browser upgrades to worry about. And you get a video recording of each test session, or just each session where tests failed.
- * **Much more!** Check out the various [contributed modules](https://github.com/testcontainers) or create your own custom container classes using [`GenericContainer`](usage/generic_containers.md) as a base.
+## Prerequisites
+
+* Docker - please see [General Docker requirements](supported_docker_environment/index.md)
+* A supported JVM testing framework:
+    * [JUnit 4](test_framework_integration/junit_4.md) - See the [JUnit 4 Quickstart Guide](quickstart/junit_4_quickstart.md)
+    * [Jupiter/JUnit 5](test_framework_integration/junit_5.md)
+    * [Spock](test_framework_integration/spock.md)
+    * *Or* manually add code to control the container/test lifecycle (See [hints for this approach](test_framework_integration/junit_4.md#manually-controlling-container-lifecycle)
+
+## Maven dependencies
+
+Testcontainers is distributed as separate JARs with a common version number:
+
+* A core JAR file for core functionality, generic containers and docker-compose support
+* A separate JAR file for each of the specialised modules. Each module's documentation describes the Maven/Gradle dependency to add to your project's build.
+
+For the core library, the latest Maven/Gradle dependency is as follows: 
+
+```groovy tab='Gradle'
+testRuntime "org.testcontainers:testcontainers:{{latest_version}}"
```

Comment:
Sorry for not spotting earlier. Should be `testCompile`

Reply:
Good spot - will change throughout.
Not sure how that got in there 😕